lib/core/src/lib/viewer/components/viewer-render.component.ts

```diff
@@ -175,6 +179,10 @@ export class ViewerRenderComponent implements OnChanges, OnInit, OnDestroy {
         }
     }
 
+    onSavingFile(isSaving: boolean) {
```

Comment:
This function is redundant probably as it only wraps this.isSaving.emit which can be called directly in html like: 
(isSaving)="isSaving.emit($event)"